src/parameter_declarations.ts

```diff
@@ -5,17 +5,20 @@ export const SeedPhraseParameter = 'seedPhrase';
 export const DrivePasswordParameter = 'drivePassword';
 export const DriveNameParameter = 'driveName';
 export const DriveKeyParameter = 'driveKey';
-export const DriveAddressParameter = 'driveAddress';
+export const AddressParameter = 'address';
 export const DriveIdParameter = 'driveId';
 export const ArAmountParameter = 'arAmount';
-export const DestinationAddressParameter = 'destAddres';
+export const DestinationAddressParameter = 'destAddress';
 export const TransactionIdParameter = 'txId';
 export const ConfirmationsParameter = 'confirmations';
+export const FolderIdParameter = 'folderId';
 export const ParentFolderIdParameter = 'parentFolderId';
 export const LocalFilePathParameter = 'localFilePath';
 export const DestinationFileNameParameter = 'destFileName';
 export const LocalFilesParameter = 'localFiles';
 export const GetAllRevisionsParameter = 'getAllRevisions';
+export const AllParameter = 'all';
```

Comment:
Let's add a task ticket in the epic to implement these.

Reply:
The new ticket: https://ardrive.atlassian.net/browse/PE-518